pyproject.toml

```diff
@@ -37,8 +38,20 @@ requires = ["poetry-core>=1.6.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.scripts]
-ci_test = "test_commands:run_tests"
-format = "test_commands:run_formatter"
-check_format = "test_commands:check_format"
-lint = "test_commands:run_linter"
-generate_docs = "test_commands:generate_docs"
+ci_test = "poetry_scripts:run_tests"
+format = "poetry_scripts:run_formatter"
+lint = "poetry_scripts:run_linter"
+fix_lint = "poetry_scripts:run_linter_fix"
+generate_docs = "poetry_scripts:generate_docs"
+update_client = "poetry_scripts:update_client"
+
+[tool.ruff]
+select = ["F", "I"]
+line-length = 100
+
+[tool.ruff.lint]
+select = ["F", "I", "E"]
+ignore = []
+
+[tool.ruff.extend-per-file-ignores]
+"jellyfish/_openapi_client/**" = ["E501"]
```

Comment:
Ignore line length for autogenerated files - some comments are too long and cannot be automatically wrapped.